python-libraries/nanover-ase/tests/openmm_ase/test_ase_runner.py

```diff
@@ -61,7 +61,7 @@ def logging_params(tmp_path):
     return params
 
 
-@pytest.fixture(params=(ASEOpenMMRunner, OpenMMIMDRunner))
+@pytest.fixture(params=(ASEOpenMMRunner,))
```

Comment:
We do not need the parametrized fixture anymore. We can get rid of `runner_class` and change `runner` to directly call `ASEOpenMMRunner`.